source/maintain_project/review_content/index.html.md.erb

```diff
@@ -0,0 +1,45 @@
+---
+title: Review content regularly
+weight: 30
+last_reviewed_on: 2019-12-04
+review_in: 4 months
+---
+
+# Review content regularly
+
+Make it easier to keep documentation up to date by configuring the [page expiry notifier][daniel-github], Daniel the Manual Spaniel.
```

Comment:
Not sure if the GitHub link helps users in the intro. I would probably remove it.